lms/djangoapps/certificates/apis/urls.py

```diff
@@ -0,0 +1,7 @@
+""" Certificates API URLs. """
+from django.conf.urls import patterns, url, include
+
+urlpatterns = patterns(
+    '',
+    url(r'^v0/', include('lms.djangoapps.certificates.apis.v0.urls', namespace='v0')),
```

Comment:
no they do not: the namespacing is defined as `certificates_api:v0:certificates:detail`